app/cafe.py

```diff
@@ -0,0 +1,24 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if (not visitor.get("vaccine")
```

Comment:
`if (not visitor.get("vaccine")` is enough here it - read the docs about  what does `.get` return